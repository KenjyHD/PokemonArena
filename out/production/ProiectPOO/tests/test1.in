Nume|Varsta|NumePokemon1;NumePokemon2;NumePokemon3|Obiect1Pokemon1;Obiect2Pokemon1;Obiect3Pokemon1|Obiect1Pokemon2;Obiect2Pokemon2;Obiect3Pokemon2|Obiect1Pokemon3;Obiect2Pokemon3;Obiect3Pokemon3
Oleg|23|Snorlax;Pikachu;Eevee|Scut;Vestă;Pelerină|Săbiuță;Baghetă Magică;Vitamine|Pelerină;Brad de Crăciun;Scut
Vasile|26|Vulpix;Psyduck;Charmander|Baghetă Magică;Vitamine;Pelerină|Vestă;Brad de Crăciun;Vitamine|Pelerină;Vestă;Săbiuță