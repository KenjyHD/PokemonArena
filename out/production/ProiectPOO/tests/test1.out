****** In arena intra Oleg si Vasile ******
############################################
###########  Luptele semifinale  ###########
############################################

------ Se lupta Snorlax cu Vulpix ------
Snorlax {HP[62], AN[3], Def[6], SDef[4]}
Vulpix {HP[36], AN[5], Def[2], SDef[4]}
--- Caracteristici cu obiecte echipate ---
Snorlax {HP[72], AN[3], Def[8], SDef[9]}
Vulpix {HP[38], AN[7], Def[2], SDef[7]}

Runda 1
Snorlax HP [62]
CDs: Abilitate1 [0] / Abilitate2 [0]

Vulpix HP [36]
CDs: Abilitate1 [0] / Abilitate2 [0]

Snorlax foloseste abilitate1 -> 4
Vulpix foloseste atac normal -> 0

Runda 2
Snorlax HP [62]
CDs: Abilitate1 [5] / Abilitate2 [0]

Vulpix HP [32]
CDs: Abilitate1 [0] / Abilitate2 [0]
Stunned

Vulpix este stunned
Snorlax foloseste abilitate2 -> 0

Runda 3
Snorlax HP [62]
CDs: Abilitate1 [4] / Abilitate2 [5]
Dodged 

Vulpix HP [32]
CDs: Abilitate1 [0] / Abilitate2 [0]

Snorlax foloseste atac normal -> 1
Vulpix foloseste abilitate2 -> 2

Runda 4
Snorlax HP [60]
CDs: Abilitate1 [3] / Abilitate2 [4]

Vulpix HP [32]
CDs: Abilitate1 [0] / Abilitate2 [7]
Dodged 

Snorlax foloseste atac normal -> 1
Vulpix foloseste abilitate1 -> 8

Runda 5
Snorlax HP [52]
CDs: Abilitate1 [2] / Abilitate2 [3]
Stunned

Vulpix HP [31]
CDs: Abilitate1 [6] / Abilitate2 [6]

Snorlax este stunned
Vulpix foloseste atac normal -> 0

Runda 6
Snorlax HP [52]
CDs: Abilitate1 [1] / Abilitate2 [2]

Vulpix HP [31]
CDs: Abilitate1 [5] / Abilitate2 [5]

Snorlax foloseste atac normal -> 1
Vulpix foloseste atac normal -> 0

Runda 7
Snorlax HP [52]
CDs: Abilitate1 [0] / Abilitate2 [1]

Vulpix HP [30]
CDs: Abilitate1 [4] / Abilitate2 [4]

Snorlax foloseste atac normal -> 1
Vulpix foloseste atac normal -> 0

Runda 8
Snorlax HP [52]
CDs: Abilitate1 [0] / Abilitate2 [0]

Vulpix HP [29]
CDs: Abilitate1 [3] / Abilitate2 [3]

Snorlax foloseste atac normal -> 1
Vulpix foloseste atac normal -> 0

Runda 9
Snorlax HP [52]
CDs: Abilitate1 [0] / Abilitate2 [0]

Vulpix HP [28]
CDs: Abilitate1 [2] / Abilitate2 [2]

Snorlax foloseste abilitate2 -> 0
Vulpix foloseste atac normal -> 0

Runda 10
Snorlax HP [52]
CDs: Abilitate1 [0] / Abilitate2 [5]
Dodged 

Vulpix HP [28]
CDs: Abilitate1 [1] / Abilitate2 [1]

Snorlax foloseste abilitate1 -> 4
Vulpix foloseste atac normal -> 0

Runda 11
Snorlax HP [52]
CDs: Abilitate1 [5] / Abilitate2 [4]

Vulpix HP [24]
CDs: Abilitate1 [0] / Abilitate2 [0]
Stunned

Vulpix este stunned
Snorlax foloseste atac normal -> 1

Runda 12
Snorlax HP [52]
CDs: Abilitate1 [4] / Abilitate2 [3]

Vulpix HP [23]
CDs: Abilitate1 [0] / Abilitate2 [0]

Snorlax foloseste atac normal -> 1
Vulpix foloseste atac normal -> 0

Runda 13
Snorlax HP [52]
CDs: Abilitate1 [3] / Abilitate2 [2]

Vulpix HP [22]
CDs: Abilitate1 [0] / Abilitate2 [0]

Snorlax foloseste atac normal -> 1
Vulpix foloseste atac normal -> 0

Runda 14
Snorlax HP [52]
CDs: Abilitate1 [2] / Abilitate2 [1]

Vulpix HP [21]
CDs: Abilitate1 [0] / Abilitate2 [0]

Snorlax foloseste atac normal -> 1
Vulpix foloseste atac normal -> 0

Runda 15
Snorlax HP [52]
CDs: Abilitate1 [1] / Abilitate2 [0]

Vulpix HP [20]
CDs: Abilitate1 [0] / Abilitate2 [0]

Snorlax foloseste atac normal -> 1
Vulpix foloseste atac normal -> 0

Runda 16
Snorlax HP [52]
CDs: Abilitate1 [0] / Abilitate2 [0]

Vulpix HP [19]
CDs: Abilitate1 [0] / Abilitate2 [0]

Snorlax foloseste abilitate1 -> 4
Vulpix foloseste atac normal -> 0

Runda 17
Snorlax HP [52]
CDs: Abilitate1 [5] / Abilitate2 [0]

Vulpix HP [15]
CDs: Abilitate1 [0] / Abilitate2 [0]
Stunned

Vulpix este stunned
Snorlax foloseste abilitate2 -> 0

Runda 18
Snorlax HP [52]
CDs: Abilitate1 [4] / Abilitate2 [5]
Dodged 

Vulpix HP [15]
CDs: Abilitate1 [0] / Abilitate2 [0]

Snorlax foloseste atac normal -> 1
Vulpix foloseste atac normal -> 0

Runda 19
Snorlax HP [52]
CDs: Abilitate1 [3] / Abilitate2 [4]

Vulpix HP [14]
CDs: Abilitate1 [0] / Abilitate2 [0]

Snorlax foloseste atac normal -> 1
Vulpix foloseste abilitate2 -> 2

Runda 20
Snorlax HP [50]
CDs: Abilitate1 [2] / Abilitate2 [3]

Vulpix HP [14]
CDs: Abilitate1 [0] / Abilitate2 [7]
Dodged 

Snorlax foloseste atac normal -> 1
Vulpix foloseste abilitate1 -> 8

Runda 21
Snorlax HP [42]
CDs: Abilitate1 [1] / Abilitate2 [2]
Stunned

Vulpix HP [13]
CDs: Abilitate1 [6] / Abilitate2 [6]

Snorlax este stunned
Vulpix foloseste atac normal -> 0

Runda 22
Snorlax HP [42]
CDs: Abilitate1 [0] / Abilitate2 [1]

Vulpix HP [13]
CDs: Abilitate1 [5] / Abilitate2 [5]

Snorlax foloseste atac normal -> 1
Vulpix foloseste atac normal -> 0

Runda 23
Snorlax HP [42]
CDs: Abilitate1 [0] / Abilitate2 [0]

Vulpix HP [12]
CDs: Abilitate1 [4] / Abilitate2 [4]

Snorlax foloseste abilitate1 -> 4
Vulpix foloseste atac normal -> 0

Runda 24
Snorlax HP [42]
CDs: Abilitate1 [5] / Abilitate2 [0]

Vulpix HP [8]
CDs: Abilitate1 [3] / Abilitate2 [3]
Stunned

Vulpix este stunned
Snorlax foloseste abilitate2 -> 0

Runda 25
Snorlax HP [42]
CDs: Abilitate1 [4] / Abilitate2 [5]
Dodged 

Vulpix HP [8]
CDs: Abilitate1 [2] / Abilitate2 [2]

Snorlax foloseste atac normal -> 1
Vulpix foloseste atac normal -> 0

Runda 26
Snorlax HP [42]
CDs: Abilitate1 [3] / Abilitate2 [4]

Vulpix HP [7]
CDs: Abilitate1 [1] / Abilitate2 [1]

Snorlax foloseste atac normal -> 1
Vulpix foloseste atac normal -> 0

Runda 27
Snorlax HP [42]
CDs: Abilitate1 [2] / Abilitate2 [3]

Vulpix HP [6]
CDs: Abilitate1 [0] / Abilitate2 [0]

Snorlax foloseste atac normal -> 1
Vulpix foloseste atac normal -> 0

Runda 28
Snorlax HP [42]
CDs: Abilitate1 [1] / Abilitate2 [2]

Vulpix HP [5]
CDs: Abilitate1 [0] / Abilitate2 [0]

Snorlax foloseste atac normal -> 1
Vulpix foloseste atac normal -> 0

Runda 29
Snorlax HP [42]
CDs: Abilitate1 [0] / Abilitate2 [1]

Vulpix HP [4]
CDs: Abilitate1 [0] / Abilitate2 [0]

Snorlax foloseste atac normal -> 1
Vulpix foloseste abilitate2 -> 2

Runda 30
Snorlax HP [40]
CDs: Abilitate1 [0] / Abilitate2 [0]

Vulpix HP [4]
CDs: Abilitate1 [0] / Abilitate2 [7]
Dodged 

Snorlax foloseste abilitate1 -> 4
Vulpix foloseste atac normal -> 0

---------> A castigat Snorlax <---------
Snorlax {HP[62], AN[3], Def[6], SDef[4]}
				  V
Snorlax {HP[63], AN[4], Def[7], SDef[5]}


------ Se lupta Pikachu cu Psyduck ------
Pikachu {HP[35], AS[4], Def[2], SDef[3]}
Psyduck {HP[43], AN[3], Def[3], SDef[3]}
--- Caracteristici cu obiecte echipate ---
Pikachu {HP[37], AS[9], Def[2], SDef[3]}
Psyduck {HP[55], AN[8], Def[4], SDef[3]}

Runda 1
Pikachu HP [35]
CDs: Abilitate1 [0] / Abilitate2 [0]

Psyduck HP [43]
CDs: Abilitate1 [0] / Abilitate2 [0]

Pikachu foloseste abilitate1 -> 6
Psyduck foloseste abilitate2 -> 2

Runda 2
Pikachu HP [33]
CDs: Abilitate1 [4] / Abilitate2 [0]
Stunned

Psyduck HP [37]
CDs: Abilitate1 [0] / Abilitate2 [5]

Pikachu este stunned
Psyduck foloseste atac normal -> 6

Runda 3
Pikachu HP [27]
CDs: Abilitate1 [3] / Abilitate2 [0]

Psyduck HP [37]
CDs: Abilitate1 [0] / Abilitate2 [4]

Pikachu foloseste abilitate2 -> 4
Psyduck foloseste atac normal -> 6

Runda 4
Pikachu HP [27]
CDs: Abilitate1 [2] / Abilitate2 [5]
Dodged 

Psyduck HP [33]
CDs: Abilitate1 [0] / Abilitate2 [3]
Stunned

Psyduck este stunned
Pikachu foloseste atac special -> 6

Runda 5
Pikachu HP [27]
CDs: Abilitate1 [1] / Abilitate2 [4]

Psyduck HP [27]
CDs: Abilitate1 [0] / Abilitate2 [2]

Pikachu foloseste atac special -> 6
Psyduck foloseste abilitate1 -> 2

Runda 6
Pikachu HP [25]
CDs: Abilitate1 [0] / Abilitate2 [3]

Psyduck HP [21]
CDs: Abilitate1 [4] / Abilitate2 [1]

Pikachu foloseste atac special -> 6
Psyduck foloseste atac normal -> 6

Runda 7
Pikachu HP [19]
CDs: Abilitate1 [0] / Abilitate2 [2]

Psyduck HP [15]
CDs: Abilitate1 [3] / Abilitate2 [0]

Pikachu foloseste abilitate1 -> 6
Psyduck foloseste abilitate2 -> 2

Runda 8
Pikachu HP [17]
CDs: Abilitate1 [4] / Abilitate2 [1]
Stunned

Psyduck HP [9]
CDs: Abilitate1 [2] / Abilitate2 [5]

Pikachu este stunned
Psyduck foloseste atac normal -> 6

Runda 9
Pikachu HP [11]
CDs: Abilitate1 [3] / Abilitate2 [0]

Psyduck HP [9]
CDs: Abilitate1 [1] / Abilitate2 [4]

Pikachu foloseste abilitate2 -> 4
Psyduck foloseste atac normal -> 6

Runda 10
Pikachu HP [11]
CDs: Abilitate1 [2] / Abilitate2 [5]
Dodged 

Psyduck HP [5]
CDs: Abilitate1 [0] / Abilitate2 [3]
Stunned

Psyduck este stunned
Pikachu foloseste atac special -> 6

---------> A castigat Pikachu <---------
Pikachu {HP[35], AS[4], Def[2], SDef[3]}
				  V
Pikachu {HP[36], AS[5], Def[3], SDef[4]}


------ Se lupta Eevee cu Charmander ------
Eevee {HP[39], AS[4], Def[3], SDef[3]}
Charmander {HP[50], AN[4], Def[3], SDef[2]}
--- Caracteristici cu obiecte echipate ---
Eevee {HP[39], AS[4], Def[6], SDef[8]}
Charmander {HP[60], AN[7], Def[3], SDef[5]}

Runda 1
Eevee HP [39]
CDs: Abilitate1 [0] / Abilitate2 [0]

Charmander HP [50]
CDs: Abilitate1 [0] / Abilitate2 [0]

Eevee foloseste abilitate1 -> 5
Charmander foloseste abilitate1 -> 4

Runda 2
Eevee HP [35]
CDs: Abilitate1 [3] / Abilitate2 [0]
Stunned

Charmander HP [45]
CDs: Abilitate1 [4] / Abilitate2 [0]

Eevee este stunned
Charmander foloseste atac normal -> 1

Runda 3
Eevee HP [34]
CDs: Abilitate1 [2] / Abilitate2 [0]

Charmander HP [45]
CDs: Abilitate1 [3] / Abilitate2 [0]

Eevee foloseste abilitate2 -> 3
Charmander foloseste abilitate2 -> 7

Runda 4
Eevee HP [27]
CDs: Abilitate1 [1] / Abilitate2 [3]

Charmander HP [42]
CDs: Abilitate1 [2] / Abilitate2 [6]
Stunned

Charmander este stunned
Eevee foloseste atac special -> 0

Runda 5
Eevee HP [27]
CDs: Abilitate1 [0] / Abilitate2 [2]

Charmander HP [42]
CDs: Abilitate1 [1] / Abilitate2 [5]

Eevee foloseste abilitate1 -> 5
Charmander foloseste atac normal -> 1

Runda 6
Eevee HP [26]
CDs: Abilitate1 [3] / Abilitate2 [1]

Charmander HP [37]
CDs: Abilitate1 [0] / Abilitate2 [4]

Eevee foloseste atac special -> 0
Charmander foloseste atac normal -> 1

Runda 7
Eevee HP [25]
CDs: Abilitate1 [2] / Abilitate2 [0]

Charmander HP [37]
CDs: Abilitate1 [0] / Abilitate2 [3]

Eevee foloseste abilitate2 -> 3
Charmander foloseste atac normal -> 1

Runda 8
Eevee HP [24]
CDs: Abilitate1 [1] / Abilitate2 [3]

Charmander HP [34]
CDs: Abilitate1 [0] / Abilitate2 [2]
Stunned

Charmander este stunned
Eevee foloseste atac special -> 0

Runda 9
Eevee HP [24]
CDs: Abilitate1 [0] / Abilitate2 [2]

Charmander HP [34]
CDs: Abilitate1 [0] / Abilitate2 [1]

Eevee foloseste abilitate1 -> 5
Charmander foloseste atac normal -> 1

Runda 10
Eevee HP [23]
CDs: Abilitate1 [3] / Abilitate2 [1]

Charmander HP [29]
CDs: Abilitate1 [0] / Abilitate2 [0]

Eevee foloseste atac special -> 0
Charmander foloseste abilitate2 -> 7

Runda 11
Eevee HP [16]
CDs: Abilitate1 [2] / Abilitate2 [0]

Charmander HP [29]
CDs: Abilitate1 [0] / Abilitate2 [6]

Eevee foloseste abilitate2 -> 3
Charmander foloseste abilitate1 -> 4

Runda 12
Eevee HP [12]
CDs: Abilitate1 [1] / Abilitate2 [3]
Stunned

Charmander HP [26]
CDs: Abilitate1 [4] / Abilitate2 [5]
Stunned

Eevee este stunned
Charmander este stunned

Runda 13
Eevee HP [12]
CDs: Abilitate1 [0] / Abilitate2 [2]

Charmander HP [26]
CDs: Abilitate1 [3] / Abilitate2 [4]

Eevee foloseste atac special -> 0
Charmander foloseste atac normal -> 1

Runda 14
Eevee HP [11]
CDs: Abilitate1 [0] / Abilitate2 [1]

Charmander HP [26]
CDs: Abilitate1 [2] / Abilitate2 [3]

Eevee foloseste atac special -> 0
Charmander foloseste atac normal -> 1

Runda 15
Eevee HP [10]
CDs: Abilitate1 [0] / Abilitate2 [0]

Charmander HP [26]
CDs: Abilitate1 [1] / Abilitate2 [2]

Eevee foloseste abilitate1 -> 5
Charmander foloseste atac normal -> 1

Runda 16
Eevee HP [9]
CDs: Abilitate1 [3] / Abilitate2 [0]

Charmander HP [21]
CDs: Abilitate1 [0] / Abilitate2 [1]

Eevee foloseste atac special -> 0
Charmander foloseste abilitate1 -> 4

Runda 17
Eevee HP [5]
CDs: Abilitate1 [2] / Abilitate2 [0]
Stunned

Charmander HP [21]
CDs: Abilitate1 [4] / Abilitate2 [0]

Eevee este stunned
Charmander foloseste abilitate2 -> 7

---------> A castigat Charmander <---------
Charmander {HP[50], AN[4], Def[3], SDef[2]}
				  V
Charmander {HP[51], AN[5], Def[4], SDef[3]}

############################################
##############  Lupta finala  ##############
############################################

------ Se lupta Snorlax cu Charmander ------
Snorlax {HP[63], AN[4], Def[7], SDef[5]}
Charmander {HP[51], AN[5], Def[4], SDef[3]}
--- Caracteristici cu obiecte echipate ---
Snorlax {HP[73], AN[4], Def[9], SDef[10]}
Charmander {HP[61], AN[8], Def[4], SDef[6]}

Runda 1
Snorlax HP [63]
CDs: Abilitate1 [0] / Abilitate2 [0]

Charmander HP [51]
CDs: Abilitate1 [0] / Abilitate2 [0]

Snorlax foloseste abilitate1 -> 4
Charmander foloseste abilitate1 -> 4

Runda 2
Snorlax HP [59]
CDs: Abilitate1 [5] / Abilitate2 [0]
Stunned

Charmander HP [47]
CDs: Abilitate1 [4] / Abilitate2 [0]
Stunned

Snorlax este stunned
Charmander este stunned

Runda 3
Snorlax HP [59]
CDs: Abilitate1 [4] / Abilitate2 [0]

Charmander HP [47]
CDs: Abilitate1 [3] / Abilitate2 [0]

Snorlax foloseste abilitate2 -> 0
Charmander foloseste abilitate2 -> 7

Runda 4
Snorlax HP [59]
CDs: Abilitate1 [3] / Abilitate2 [5]
Dodged 

Charmander HP [47]
CDs: Abilitate1 [2] / Abilitate2 [6]

Snorlax foloseste atac normal -> 0
Charmander foloseste atac normal -> 0

Runda 5
Snorlax HP [59]
CDs: Abilitate1 [2] / Abilitate2 [4]

Charmander HP [47]
CDs: Abilitate1 [1] / Abilitate2 [5]

Snorlax foloseste atac normal -> 0
Charmander foloseste atac normal -> 0

Runda 6
Snorlax HP [59]
CDs: Abilitate1 [1] / Abilitate2 [3]

Charmander HP [47]
CDs: Abilitate1 [0] / Abilitate2 [4]

Snorlax foloseste atac normal -> 0
Charmander foloseste abilitate1 -> 4

Runda 7
Snorlax HP [55]
CDs: Abilitate1 [0] / Abilitate2 [2]
Stunned

Charmander HP [47]
CDs: Abilitate1 [4] / Abilitate2 [3]

Snorlax este stunned
Charmander foloseste atac normal -> 0

Runda 8
Snorlax HP [55]
CDs: Abilitate1 [0] / Abilitate2 [1]

Charmander HP [47]
CDs: Abilitate1 [3] / Abilitate2 [2]

Snorlax foloseste abilitate1 -> 4
Charmander foloseste atac normal -> 0

Runda 9
Snorlax HP [55]
CDs: Abilitate1 [5] / Abilitate2 [0]

Charmander HP [43]
CDs: Abilitate1 [2] / Abilitate2 [1]
Stunned

Charmander este stunned
Snorlax foloseste abilitate2 -> 0

Runda 10
Snorlax HP [55]
CDs: Abilitate1 [4] / Abilitate2 [5]
Dodged 

Charmander HP [43]
CDs: Abilitate1 [1] / Abilitate2 [0]

Snorlax foloseste atac normal -> 0
Charmander foloseste atac normal -> 0

Runda 11
Snorlax HP [55]
CDs: Abilitate1 [3] / Abilitate2 [4]

Charmander HP [43]
CDs: Abilitate1 [0] / Abilitate2 [0]

Snorlax foloseste atac normal -> 0
Charmander foloseste atac normal -> 0

Runda 12
Snorlax HP [55]
CDs: Abilitate1 [2] / Abilitate2 [3]

Charmander HP [43]
CDs: Abilitate1 [0] / Abilitate2 [0]

Snorlax foloseste atac normal -> 0
Charmander foloseste abilitate2 -> 7

Runda 13
Snorlax HP [48]
CDs: Abilitate1 [1] / Abilitate2 [2]

Charmander HP [43]
CDs: Abilitate1 [0] / Abilitate2 [6]

Snorlax foloseste atac normal -> 0
Charmander foloseste atac normal -> 0

Runda 14
Snorlax HP [48]
CDs: Abilitate1 [0] / Abilitate2 [1]

Charmander HP [43]
CDs: Abilitate1 [0] / Abilitate2 [5]

Snorlax foloseste atac normal -> 0
Charmander foloseste abilitate1 -> 4

Runda 15
Snorlax HP [44]
CDs: Abilitate1 [0] / Abilitate2 [0]
Stunned

Charmander HP [43]
CDs: Abilitate1 [4] / Abilitate2 [4]

Snorlax este stunned
Charmander foloseste atac normal -> 0

Runda 16
Snorlax HP [44]
CDs: Abilitate1 [0] / Abilitate2 [0]

Charmander HP [43]
CDs: Abilitate1 [3] / Abilitate2 [3]

Snorlax foloseste atac normal -> 0
Charmander foloseste atac normal -> 0

Runda 17
Snorlax HP [44]
CDs: Abilitate1 [0] / Abilitate2 [0]

Charmander HP [43]
CDs: Abilitate1 [2] / Abilitate2 [2]

Snorlax foloseste abilitate2 -> 0
Charmander foloseste atac normal -> 0

Runda 18
Snorlax HP [44]
CDs: Abilitate1 [0] / Abilitate2 [5]
Dodged 

Charmander HP [43]
CDs: Abilitate1 [1] / Abilitate2 [1]

Snorlax foloseste atac normal -> 0
Charmander foloseste atac normal -> 0

Runda 19
Snorlax HP [44]
CDs: Abilitate1 [0] / Abilitate2 [4]

Charmander HP [43]
CDs: Abilitate1 [0] / Abilitate2 [0]

Snorlax foloseste atac normal -> 0
Charmander foloseste abilitate1 -> 4

Runda 20
Snorlax HP [40]
CDs: Abilitate1 [0] / Abilitate2 [3]
Stunned

Charmander HP [43]
CDs: Abilitate1 [4] / Abilitate2 [0]

Snorlax este stunned
Charmander foloseste abilitate2 -> 7

Runda 21
Snorlax HP [33]
CDs: Abilitate1 [0] / Abilitate2 [2]

Charmander HP [43]
CDs: Abilitate1 [3] / Abilitate2 [6]

Snorlax foloseste atac normal -> 0
Charmander foloseste atac normal -> 0

Runda 22
Snorlax HP [33]
CDs: Abilitate1 [0] / Abilitate2 [1]

Charmander HP [43]
CDs: Abilitate1 [2] / Abilitate2 [5]

Snorlax foloseste atac normal -> 0
Charmander foloseste atac normal -> 0

Runda 23
Snorlax HP [33]
CDs: Abilitate1 [0] / Abilitate2 [0]

Charmander HP [43]
CDs: Abilitate1 [1] / Abilitate2 [4]

Snorlax foloseste abilitate2 -> 0
Charmander foloseste atac normal -> 0

Runda 24
Snorlax HP [33]
CDs: Abilitate1 [0] / Abilitate2 [5]
Dodged 

Charmander HP [43]
CDs: Abilitate1 [0] / Abilitate2 [3]

Snorlax foloseste abilitate1 -> 4
Charmander foloseste abilitate1 -> 4

Runda 25
Snorlax HP [29]
CDs: Abilitate1 [5] / Abilitate2 [4]
Stunned

Charmander HP [39]
CDs: Abilitate1 [4] / Abilitate2 [2]
Stunned

Snorlax este stunned
Charmander este stunned

Runda 26
Snorlax HP [29]
CDs: Abilitate1 [4] / Abilitate2 [3]

Charmander HP [39]
CDs: Abilitate1 [3] / Abilitate2 [1]

Snorlax foloseste atac normal -> 0
Charmander foloseste atac normal -> 0

Runda 27
Snorlax HP [29]
CDs: Abilitate1 [3] / Abilitate2 [2]

Charmander HP [39]
CDs: Abilitate1 [2] / Abilitate2 [0]

Snorlax foloseste atac normal -> 0
Charmander foloseste atac normal -> 0

Runda 28
Snorlax HP [29]
CDs: Abilitate1 [2] / Abilitate2 [1]

Charmander HP [39]
CDs: Abilitate1 [1] / Abilitate2 [0]

Snorlax foloseste atac normal -> 0
Charmander foloseste abilitate2 -> 7

Runda 29
Snorlax HP [22]
CDs: Abilitate1 [1] / Abilitate2 [0]

Charmander HP [39]
CDs: Abilitate1 [0] / Abilitate2 [6]

Snorlax foloseste atac normal -> 0
Charmander foloseste abilitate1 -> 4

Runda 30
Snorlax HP [18]
CDs: Abilitate1 [0] / Abilitate2 [0]
Stunned

Charmander HP [39]
CDs: Abilitate1 [4] / Abilitate2 [5]

Snorlax este stunned
Charmander foloseste atac normal -> 0

Runda 31
Snorlax HP [18]
CDs: Abilitate1 [0] / Abilitate2 [0]

Charmander HP [39]
CDs: Abilitate1 [3] / Abilitate2 [4]

Snorlax foloseste abilitate2 -> 0
Charmander foloseste atac normal -> 0

Runda 32
Snorlax HP [18]
CDs: Abilitate1 [0] / Abilitate2 [5]
Dodged 

Charmander HP [39]
CDs: Abilitate1 [2] / Abilitate2 [3]

Snorlax foloseste abilitate1 -> 4
Charmander foloseste atac normal -> 0

Runda 33
Snorlax HP [18]
CDs: Abilitate1 [5] / Abilitate2 [4]

Charmander HP [35]
CDs: Abilitate1 [1] / Abilitate2 [2]
Stunned

Charmander este stunned
Snorlax foloseste atac normal -> 0

Runda 34
Snorlax HP [18]
CDs: Abilitate1 [4] / Abilitate2 [3]

Charmander HP [35]
CDs: Abilitate1 [0] / Abilitate2 [1]

Snorlax foloseste atac normal -> 0
Charmander foloseste abilitate1 -> 4

Runda 35
Snorlax HP [14]
CDs: Abilitate1 [3] / Abilitate2 [2]
Stunned

Charmander HP [35]
CDs: Abilitate1 [4] / Abilitate2 [0]

Snorlax este stunned
Charmander foloseste abilitate2 -> 7

Runda 36
Snorlax HP [7]
CDs: Abilitate1 [2] / Abilitate2 [1]

Charmander HP [35]
CDs: Abilitate1 [3] / Abilitate2 [6]

Snorlax foloseste atac normal -> 0
Charmander foloseste atac normal -> 0

Runda 37
Snorlax HP [7]
CDs: Abilitate1 [1] / Abilitate2 [0]

Charmander HP [35]
CDs: Abilitate1 [2] / Abilitate2 [5]

Snorlax foloseste abilitate2 -> 0
Charmander foloseste atac normal -> 0

Runda 38
Snorlax HP [7]
CDs: Abilitate1 [0] / Abilitate2 [5]
Dodged 

Charmander HP [35]
CDs: Abilitate1 [1] / Abilitate2 [4]

Snorlax foloseste atac normal -> 0
Charmander foloseste atac normal -> 0

Runda 39
Snorlax HP [7]
CDs: Abilitate1 [0] / Abilitate2 [4]

Charmander HP [35]
CDs: Abilitate1 [0] / Abilitate2 [3]

Snorlax foloseste abilitate1 -> 4
Charmander foloseste atac normal -> 0

Runda 40
Snorlax HP [7]
CDs: Abilitate1 [5] / Abilitate2 [3]

Charmander HP [31]
CDs: Abilitate1 [0] / Abilitate2 [2]
Stunned

Charmander este stunned
Snorlax foloseste atac normal -> 0

Runda 41
Snorlax HP [7]
CDs: Abilitate1 [4] / Abilitate2 [2]

Charmander HP [31]
CDs: Abilitate1 [0] / Abilitate2 [1]

Snorlax foloseste atac normal -> 0
Charmander foloseste atac normal -> 0

Runda 42
Snorlax HP [7]
CDs: Abilitate1 [3] / Abilitate2 [1]

Charmander HP [31]
CDs: Abilitate1 [0] / Abilitate2 [0]

Snorlax foloseste atac normal -> 0
Charmander foloseste abilitate1 -> 4

Runda 43
Snorlax HP [3]
CDs: Abilitate1 [2] / Abilitate2 [0]
Stunned

Charmander HP [31]
CDs: Abilitate1 [4] / Abilitate2 [0]

Snorlax este stunned
Charmander foloseste abilitate2 -> 7

---------> A castigat Charmander <---------
Charmander {HP[51], AN[5], Def[4], SDef[3]}
				  V
Charmander {HP[52], AN[6], Def[5], SDef[4]}

##  Vasile a iesit castigator din arena  ##
